### PR TITLE
Expose an initializer function for non-abstract contracts in their upgradeable versions

### LINKED ISSUE #5002: Mark AccessManagerUpgradeable abstract or implement initializer function
**🧐 Motivation**
I want to use AccessManagerUpgradeable the same way I can use AccessManager contract.

**📝 Details**
It is impossible to use AccessManagerUpgradeable as is right know. It lacks initializer function, so it is not possible to deploy it correctly. And also it is not marked abstract to force implement initializer function in sub-contract.

### PR DESCRIPTION
**Pull request by ernestognw**
Fixes #5002 

#### PR Checklist

- [ ] Tests
- [ ] Documentation
- [ ] Changeset entry (run `npx changeset add`)

### CODE REVIEW

File: scripts/upgradeable/transpile.sh [hunk 1]
```diff
@@ -39,6 +39,10 @@ npx @openzeppelin/upgrade-safe-transpiler -D \
   -x '!contracts/proxy/utils/UUPSUpgradeable.sol' \
   -x '!contracts/proxy/beacon/IBeacon.sol' \
   -p 'contracts/**/presets/**/*' \
```
Comment (Amxx) [inline scripts/upgradeable/transpile.sh:41]: We should remove that line since we don't have any presets any more, right?
```suggestion
```

Comment (ernestognw) [reply scripts/upgradeable/transpile.sh:41]: Yeah, that's right!

**Review by Amxx — DISMISSED** [review]
LGTM

File: .changeset/thin-walls-drop.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': minor
+---
+
+`AccessManager`, `VestingWallet`, `TimelockController` and `ERC2771Forwarder`: Added an `initializer` function in their corresponding upgradeable variants.
```
Comment (Amxx) [inline .changeset/thin-walls-drop.md:5]:
```suggestion
`AccessManager`, `VestingWallet`, `TimelockController` and `ERC2771Forwarder`: Added a public `initializer` function in their corresponding upgradeable variants.
```

**Review by Amxx — APPROVED** [review]

**Pull request merged**